src/topkg_build.ml

```diff
@@ -33,11 +38,16 @@ let with_dir b dir = { b with dir }
 let nop = fun _ -> Ok ()
 
 let cmd c os files =
-  Topkg_os.Cmd.run @@ Topkg_cmd.(build_cmd c os %% of_list files)
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:false c os %% v "-just-plugin") >>= fun () ->
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:true c os %% of_list files)
```

Comment:
@dbuenzli There is no need to separately invoke -just-plugin anymore because https://github.com/ocaml/ocamlbuild/pull/130#event-899198397 got merged. I will update the PR shortly.